extensions/amp-tiktok/0.1/test/validator-amp-tiktok.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-tiktok example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-tiktok {
+      color: red;
+    }
+  </style>
+  <script async custom-element="amp-tiktok" src="https://cdn.ampproject.org/v0/amp-tiktok-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <!-- Valid `amp-tiktok` components -->
+  <amp-tiktok width="500" height="500" data-src="6943753342808034566">
+  </amp-tiktok>
+  <amp-tiktok
+    width="700"
+    height="800"
+    data-src="https://www.tiktok.com/@scout2015/video/6943753342808034566"
+  >
+  </amp-tiktok>
+  <amp-tiktok width="500" height="800">
```

Comment:
Is this valid or invalid? Please add a comment here. If it's invalid then please state why it is. This should apply to the responses below.